jina/types/document/uid.py

```diff
@@ -31,7 +31,7 @@
 from jina.logging import default_logger
 
 _digest_size = 8
-_id_regex = re.compile(r'[0-9a-fA-F]{16}')
+_id_regex = re.compile(r'[0-9a-fA-F]+')
```

Comment:
If we do that, the number of characters must be a multiple of 2, since we hexlify them at some point. Could you adapt the regex please?

Reply:
Sure, no problem